vignettes/projecting_incidence.Rmd

```diff
@@ -92,23 +92,29 @@ t0 <- rep(days_since_index, seed_cases$cases)
 t0
 ```
 
-### Serial interval
+### Generation time
 
-Next, we will set up the serial interval distribution, that is, the time
-between successive onsets of cases in a transmission chain.
-
-The log-normal distribution is commonly used in epidemiology to characterise 
-quantities such as the serial interval because it has a large variance 
-and can only be positive-valued [@nishiura2007; @limpert2001]. 
+In epidemiology, the generation interval is the duration between successive
+infectious events in a chain of transmission. Similarly, the serial
+interval is the duration between observed symptom onset times between
+successive cases in a transmission chain. The generation interval is
+often hard to observe because exact times of infection are hard to
+measure hence, the serial interval is often used instead. Here, we
+use the serial interval to represent what would normally be called the
+generation interval, that is, the time between successive cases.
```

Comment:
```suggestion
use the serial interval and interpret the simulated case data to represent
symptom onset.
```

I wasn't quite sure what you meant so attempted to rephrase - does this make sense?